test/javascripts/lib/checklist-test.js

```diff
@@ -15,7 +15,7 @@ async function prepare(raw) {
   const decoratorHelper = { getModel: () => model };
 
   const $elem = $(cooked.string);
-  checklistSyntax($elem, decoratorHelper);
+  checklistSyntax($elem[0], decoratorHelper);
```

Comment:
This broke the tests 🙂 you were passing only whatever the first element happened to be 😛 


This works:

```suggestion
  const $elem = $(`<div>${cooked.string}</div>`);
  checklistSyntax($elem[0], decoratorHelper);
```